diff --git a/node_modules/vitest/index.cjs b/node_modules/vitest/index.cjs
index 85fc303..ed32705 100644
--- a/node_modules/vitest/index.cjs
+++ b/node_modules/vitest/index.cjs
@@ -1,5 +1,6 @@
+console.log("vitest/index.cjs is called");
 throw new Error(
-  'Vitest cannot be imported in a CommonJS module using require(). Please use "import" instead.'
-  + '\n\nIf you are using "import" in your source code, then it\'s possible it was bundled into require() automatically by your bundler. '
-  + 'In that case, do not bundle CommonJS output since it will never work with Vitest, or use dynamic import() which is available in all CommonJS modules.',
-)
+  'Vitest cannot be imported in a CommonJS module using require(). Please use "import" instead.' +
+    '\n\nIf you are using "import" in your source code, then it\'s possible it was bundled into require() automatically by your bundler. ' +
+    "In that case, do not bundle CommonJS output since it will never work with Vitest, or use dynamic import() which is available in all CommonJS modules.",
+);
